espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityHelpers.kt

```diff
@@ -87,6 +88,8 @@ object ActivityHelpers {
             AndroidLogger.logger.info("Staring activity with custom options: $intentOptions")
             makeIntent(intentOptions)
         }
-        instrumentation.startActivitySync(intent)
+        val options = ActivityOptions.makeBasic()
+        if (displayId != null) options.launchDisplayId = displayId.toInt()
```

Comment:
Only for API level 26+?
https://developer.android.com/reference/kotlin/android/app/ActivityOptions#setlaunchdisplayid

Reply:
like https://github.com/appium/appium-espresso-driver/blob/d7b3f6cf27ec75741041c033d83728b6fcd01f96/espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/AutofillHelpers.kt#L40

Reply:
You can also write like below:
```kotlin
displayId?.let {
    options.launchDisplayId = it.toInt()
}
```

Reply:
Yes. It is 26+, let me add this